src/api.js

```diff
@@ -21,3 +21,9 @@ export const getComments = (article_id) => {
     return data;
   });
 };
+
+export const updateVoteCount = (article_id, vote) => {
+  return api.patch(`/articles/${article_id}`, vote).then(({ data }) => {
+    return data;
+  });
+};
```

Comment:
Nice, but instead of sending back the data object, we should send back the specific data from this object that we actually want on the other side